### PR TITLE
Update to lodash-cli v3.1.1

### PR DESCRIPTION
**Pull request by mathiasbynens**

### CODE REVIEW

**Comment by jdalton** [comment]
:tada:

**Comment by d10** [comment]
Thanks, @mathiasbynens! I’ll pull this in tomorrow.

File: server.js [hunk 1]
```diff
@@ -72,7 +72,7 @@
     }
 
     // minify?
-    args.push(query.minify == 'true' ? '--minify' : '--debug')
+    args.push(query.minify == 'true' ? '--production' : '--debug')
```
Comment (jdalton) [inline server.js]: `--debug` is now `--development`

File: public/index.html [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-  <meta name="viewport" content="width=device-width">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
   <link rel="stylesheet" href="css/normalize.css">
   <link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-  <div class="outer">    
-    <h1 class="center"><a href="http://lodash.com/">Lo-Dash</a> <span>v2.4.1</span></h1>
+  <div class="outer">
+    <h1 class="center"><a href="https://lodash.com/">Lo-Dash</a> <span>v3.1.1</span></h1>
```
Comment (jdalton) [inline public/index.html]: Can the version be gotten from say lodash on the page's `_.VERSION`. Also include lodash on the page if it isn't :P.

File: public/index.html [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
     <h2 class="center">Web Builder</h2>
 
     <div class="inner">
       <p>
-        See the <a href="http://lodash.com/">Lo-Dash</a> <a href="http://lodash.com/custom-builds">custom builds</a> documentation for more information.<br>
+        See the <a href="https://lodash.com/">Lo-Dash</a> <a href="https://lodash.com/custom-builds">custom builds</a> documentation for more information.<br>
```
Comment (jdalton) [inline public/index.html]: Lo-Dash -> lodash everywhere.

File: server.js (hunk 1 above)
Comment (jdalton) [inline server.js]: The regexp [here](https://github.com/mathiasbynens/lodash-webbuilder/blob/05a0b4671a29b89554113ccddd0d7cff9e87e334/server.js#L51) needs to be adjusted to remove `underscore|backbone`

**Comment by jdalton** [comment]
Related to #5.

File: server.js [hunk 4]
```diff
@@ -46,9 +46,9 @@
   function buildLodash(req, res, query) {
     var args = [lodashCli], errors = [];
 
-    // add Lo-Dash build modifier
+    // add lodash build modifier
     var modifier = query.modifier;
-    if (modifier && /^(?:compat|modern|underscore|backbone)$/i.test(modifier)) {
+    if (modifier && /^(?:compat|modern)$/i.test(modifier)) {
```
Comment (mathiasbynens) [inline server.js]: @jdalton Like this?

File: public/index.html [hunk 5]
```diff
@@ -35,8 +35,6 @@ <h2 class="center">Web Builder</h2>
           <select name="modifier">
             <option value="compat" selected>compat</option>
             <option value="modern">modern</option>
```
Comment (mathiasbynens) [inline public/index.html]: @jdalton Removed them here too.

File: server.js (hunk 4 above)
Comment (jdalton) [inline server.js]: Yap!

**Comment by d10** [comment]
@mathiasbynens could you rebase this onto `dev` instead? Thanks! :smiley:

**Pull request closed without merging**